ionic2/app/components/poi-card/poi-card.component.html

```diff
@@ -1,46 +1,59 @@
-<div class="slide-card" #slideCard @slide="slideState">
-  <div class="poke-poi-card-content" *ngIf="pokePOI" [ngClass]="'poi-type-'+pokePOI.getType()">
-    <ion-row>
+<div id="slide-card" #slideCard @slide="slideState">
+  <!-- Sightings -->
+  <div *ngIf="poi && poi.type == 'sighting' && pokemon">
+    <section class="side-section" id="sighting-side-section">
+      <poke-poi-bubble [pokemon]="pokemon"
+                       [appearedOn]="poi.appearedOn"></poke-poi-bubble>
+      <div>
+        <poke-type [type]="type" *ngFor="let type of pokemon.types">
+        </poke-type>
+      </div>
+    </section>
 
-      <ion-col class="poke-column">
-        <ion-row>
-          <poke-poi-bubble [pokePOI]="pokePOI"></poke-poi-bubble>
-        </ion-row>
-        <ion-row>
-          <poke-type [type]="type" *ngFor="let type of pokePOI.pokemon.types">
-          </poke-type>
-        </ion-row>
-      </ion-col>
-
-      <ion-col style="flex-grow: 1;">
-        <ion-row>
-          <span class="poke-name">{{ pokePOI.pokemon.name }}</span>
-          <poke-rarity-badge [rarity]="pokePOI.pokemon.appearanceLikelihood">
-          </poke-rarity-badge>
-        </ion-row>
-        <ion-row class="poke-description">
-          {{ pokePOI.pokemon.description }}
-        </ion-row>
+    <section class="main-section" id="sighting-main-section">
+      <div class="name-and-rarity">
+        <span class="title">{{ pokemon.name }}</span>
+        <poke-rarity-badge [rarity]="pokemon.appearanceLikelihood">
+        </poke-rarity-badge>
+      </div>
+      <div class="description">{{ pokemon.description }}</div>
+      <div class="buttons">
+        <button clear (click)="showDirections()">
+          <ion-icon name="navigate"></ion-icon> Directions
+        </button>
+        <button clear (click)="launchPokeDex()">
+          <ion-icon name="book"></ion-icon> PokeDex
+        </button>
+      </div>
+    </section>
 
+    <div class="line" id="sighting-line"></div>
+    <div class="line-badge" id="sighting-line-badge">Pokemon Sighting</div>
+  </div>
 
-        <ion-row class="action-button-bar">
-          <button clear (click)="showDirections()">
-            <ion-icon name="navigate"></ion-icon>
-            Directions
-          </button>
+  <!-- Mobs -->
+  <div *ngIf="poi && poi.type == 'mob'">
+    <section class="side-section" id="mob-side-section">
+      <img src="img/pokemon-mob.svg" alt="Pokemon Logo">
+    </section>
 
-          <button clear (click)="launchPokeDex()">
-            <ion-icon name="book"></ion-icon>
-            PokeDex
-          </button>
-        </ion-row>
-      </ion-col>
-    </ion-row>
+    <section class="main-section" id="mob-main-section">
+      <div class="title">PokeMob</div>
+      <div class="description">
+        A lot of players seem to gather at this location!<br><br>
+        Based on {{ poi.tweets.length }} tweet(s).
```

Comment:
By the definition of a mob there should always be multiple tweets so I think you can replace `tweet(s)` by `tweets`